scenes/quests/lore_quests/quest_001/2_ink_combat/components/projectile/components/projectile.gd

```diff
@@ -109,8 +109,6 @@ func add_small_fx() -> void:
 
 
 func _on_body_entered(body: Node2D) -> void:
-	if body.owner is FillingBarrel and not can_hit_enemy:
-		return
```

Comment:
It might make the game easier but, I like this change 👍🏾 , also we weren't providing the player much feedback about _when_ the ink blobs were able to fill the barrels (I think the only difference was that there were more particles, right?)

Reply:
Yeah current feedback is not very noticeable, it is more particles and doubled speed of the frame-by-frame animation.